packages/sirius-web/backend/sirius-web/src/test/java/org/eclipse/sirius/web/data/TestIdentifiers.java

```diff
@@ -26,7 +26,8 @@ public final class TestIdentifiers {
     public static final String UML_SAMPLE_PROJECT = "7ba7bda7-13b9-422a-838b-e45a3597e952";
     public static final String SYSML_SAMPLE_PROJECT = "4164c661-e0cb-4071-b25d-8516440bb8e8";
     public static final UUID SYSML_IMAGE = UUID.fromString("ff37f0eb-effb-4c57-b17f-76bc7ea64f5b");
-    public static final String ECORE_SAMPLE_PROJECT = "99d336a2-3049-439a-8853-b104ffb22653";
+    public static final String ECORE_SAMPLE_PROJECT = "cb133bf0-d7aa-4a83-a277-0972919dd46a";
+    public static final String ECORE_SAMPLE_PROJECT_ID = "99d336a2-3049-439a-8853-b104ffb22653";
```

Comment:
There's no reason to rename the existing identifier to use `_ID`, it is an identifier, that's the goal of the class. Use another name if it's relevant but right now I don't know how anyone could tell the difference between the identifier `ECORE_SAMPLE_PROJECT` and `ECORE_SAMPLE_PROJECT_ID`.